src/proto/generation.proto

```diff
@@ -19,6 +20,8 @@ enum ArtifactType {
   ARTIFACT_EMBEDDING = 5;
   ARTIFACT_CLASSIFICATIONS = 6;
   ARTIFACT_MASK = 7;
+  ARTIFACT_LATENTS = 8;
```

Comment:
pretty sure this should be singular, especially if ARTIFACT_TENSOR is singular.

Reply:
:+1: